shadow-dom/nested-slot-remove-crash.html

```diff
@@ -27,7 +27,7 @@
   // The renderer should not crash here:
   root.querySelector("#outer2").remove();
 
-  const root = host.attachShadow({mode:"open"});
+  root = host.attachShadow({mode:"open"});
```

Comment:
Actually, this doesn't work either because it calls attachShadow on the same element...

Reply:
Good point, I'll fix...